budgetportal/tests/test_department.py

```diff
@@ -11,10 +11,15 @@
 from budgetportal import models
 from django.test import TestCase
 from mock import Mock
+import json
+
+with open('budgetportal/tests/test_data/budget_and_actual.json', 'r') as mock_data:
```

Comment:
nice - I've been thinking and googled a bit around conventions for this - did you just decide to do it this way or is this a convention you found somewherE?